tests/VolleyManagement.UnitTests/Services/TournamentRequestService/TournamentRequestServiceTests.cs

```diff
@@ -330,10 +329,9 @@ private void MockGetAllTournamentRequestQuery(TournamentRequest testData)
             _getRequestByAllQueryMock.Setup(tr => tr.Execute(It.IsAny<FindByTeamTournamentCriteria>())).Returns(testData);
         }
 
-        private void VerifyAddedTeam(int requestId, int tournamentId, int groupId, Times times, Times unitOfWorkTimes)
+        private void VerifyAddedTeam(Times times)
```

Comment:
Method name does not correspond to its meaning. Current name suggests team is added to db.